src/resources/js/v2/__tests__/tickets-utils.test.js

```diff
@@ -0,0 +1,117 @@
+global.TribeCartEndpoint = { url : 'test' };
```

Comment:
There are a number of small formatting things in this file that may be causing eslint issues.